packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1442,6 +1485,8 @@ export class ContainerRuntime
 				disableChunking,
 				disableAttachReorder: this.disableAttachReorder,
 				disablePartialFlush,
+				idCompressorEnabled: compressorEnabled ? true : undefined,
```

Comment:
Isn't `idCompressorEnabled: compressorEnabled` enough? If it is explicitly set to false, you want false to be logged.